pkg/virt-handler/device-manager/generic_device.go

```diff
@@ -340,3 +345,17 @@ func (dpi *GenericDevicePlugin) healthCheck() error {
 func SocketPath(deviceName string) string {
 	return filepath.Join(pluginapi.DevicePluginPath, fmt.Sprintf("kubevirt-%s.sock", deviceName))
 }
+
+func (dpi *GenericDevicePlugin) Started() bool {
+	return dpi.started
+}
+
+func (dpi *GenericDevicePlugin) Healthy() bool {
+	for _, dev := range dpi.devs {
+		if dev.Health != pluginapi.Healthy {
```

Comment:
This seems to indicate that we would only ever set the node to schedulable if the kvm device is actually present. I think what we want is just a check that we could talk to the kubelet successfully.